plutus-metatheory/src/Type/Equality.lagda.md

```diff
@@ -33,6 +33,13 @@ open import Type.RenamingSubstitution
 
 This serves as a declaritive specification of the semantics of types.
 
+We define type equality as an intrinsically scoped and kinded relation. In particular,
+this means it is impossible to state an equation between types in different contexts, or of
+different kinds. The only interesting rule is the β-rule from the lambda calculus. We omit
```

Comment:
Is this just for convenience? Surely types can be equal in different contexts (but not of different kinds)?

Reply:
You could define it more heterogeneously and allow more degrees of freedom such as different contexts and kinds but it's not necessary.